tcms/testruns/tests/test_admin.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+
+
+from django.urls import reverse
+
+from tcms.tests import LoggedInTestCase
+
+
+class TestTestRunAdmin(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.tester.is_staff = True
+        cls.tester.save()
```

Comment:
Try using `initiate_user_with_default_setups` which also sets is_staff. This will give the currently logged in user the permissions to add/delete common objects.